resources/views/models/view.blade.php

```diff
@@ -145,9 +145,9 @@ class="table table-striped snipe-table"
                                         <tr>
                                             <td><i class="{{ Helper::filetype_icon($file->filename) }} icon-med" aria-hidden="true"></i></td>
                                             <td>
-                                                @if ( Helper::checkUploadIsImage($file->get_src('assets')))
-                                                    <a href="{{ route('show/modelFile', ['modelId' => $model->id, 'fileId' =>$file->id]) }}" data-toggle="lightbox" data-type="image" data-title="{{ $file->filename }}" data-footer="{{ Helper::getFormattedDateObject($asset->last_checkout, 'datetime', false) }}">
-                                                        <img src="{{ route('show/modelfile', ['assetId' => $model->id, 'fileId' =>$file->id]) }}" style="max-width: 50px;">
+                                                @if ( Helper::checkUploadIsImage($file->get_src('assetmodels')))
+                                                    <a href="{{ route('show/modelfile', ['modelID' => $model->id, 'fileId' => $file->id]) }}" data-toggle="lightbox" data-type="image" data-title="{{ $file->filename }}">
```

Comment:
Also `modelFile` seems to be changing to `modelfile` - and I don't see any other corresponding change. Though, if we only ever refer to the route using some kind of `route()` helper that should be fine?

Reply:
Yeah, I legit don't know how that ever worked for models lol

It's lowercase in the routes file.